Library/Homebrew/test/dev-cmd/audit_spec.rb

```diff
@@ -58,13 +58,20 @@ class #{Formulary.class_s(name)} < Formula
   end
 
   describe FormulaAuditor do
-    def formula_auditor(name, text, options = {})
-      path = Pathname.new "#{dir}/#{name}.rb"
-      path.open("w") do |f|
-        f.write text
+    def formula_auditor(name_or_formula, text = nil, options = {})
+      formula = case name_or_formula
+      when String
+        path = Pathname.new "#{dir}/#{name_or_formula}.rb"
+        path.open("w") do |f|
+          f.write text
+        end
+
+        Formulary.factory(path)
+      when Formula
+        name_or_formula
```

Comment:
Can we just use `described_class.new` in tests instead of calling this `formula_auditor` method in these cases? That way the method doesn't have to change or have this `case`

Reply:
Ah, you're right, thanks again!
Sorry for the noise